manifest.xml

```diff
@@ -12,4 +12,4 @@
 		<type>elgg_release</type>
 		<version>1.9</version>
 	</requires>
-</plugin_manifest>
\ No newline at end of file
+</plugin_manifest>
```

Comment:
No need for changing this file.

Reply:
Same issue as above, my editor had changed the end of line chars